src/components/Register.vue

```diff
@@ -110,12 +115,14 @@ const handleRegister = async () => {
   } catch (err) {
     console.error('Registration error:', err);
     error.value = 'Registration failed. Please try again.';
-  } finally {
+  }
+  finally {
+
     isLoading.value = false;
     setTimeout(() => {
       router.push('/');
     }, 1000);
```

Comment:
What is going on here?

Reply:
Upon registering, the page will wait one second then push to the main home page at the ref `/`. If I'm not mistaken the one second wait was @jasgcode 's idea so that it would indicate to the user that something is happening and the registration was successful.